blottertrax/description_provider.py

```diff
@@ -0,0 +1,71 @@
+import musicbrainzngs
+
+from blottertrax.config import Config
+from blottertrax.exceptions.description_exception import DescriptionException
+from blottertrax.helper import templates
+from blottertrax.value_objects.parsed_submission import ParsedSubmission
+
+
+class DescriptionProvider:
+    musicbrainz = None
+
+    def __init__(self):
+        config = Config()
+
+        self.musicbrainz = musicbrainzngs
+        self.musicbrainz.auth(config.MUSICBRAINZ_USER, config.MUSICBRAINZ_PASSWORD)
+        self.musicbrainz.set_useragent("BlotterTrax", "0.1", "https://github.com/martijnboers/BlotterTrax")
+
+    def get_reply(self, parsed_submission: ParsedSubmission) -> str:
+        if parsed_submission.success is False:
+            raise DescriptionException('DescriptionProvider requires the submission title')
+
+        query = '"{}" AND artist:"{}"'.format(parsed_submission.track_title, parsed_submission.artist)
+        result = self.musicbrainz.search_recordings(query=query, limit=1, )
+
+        if result['recording-count'] == 0:
+            raise DescriptionException('No recordings found for artist')
+
+        recording = result['recording-list'][0]
+        artist_ids = list(map(lambda a: a['artist']['id'], recording['artist-credit']))
+        artists = list(map(lambda a: self._get_artist_by_id(a), artist_ids))
```

Comment:
Looks like we are only concerned with the first result here.  Maybe _get_artist_by_id should just return the first?  Or instead of this being artists just grab the first one and call the variable artist.  Keeps us from having the [0] nomenclature everywhere.

Reply:
Was thinking about showing other credited artists, but since I didn't include it in this PR removing that